src/python-fastui/fastui/json_schema.py

```diff
@@ -154,7 +154,8 @@ def json_schema_obj_to_fields(
     required = set(schema.get('required', []))
```

Comment:
I'm not sure I understand, because this is what PR already implements: bool fields are not required. Maybe you mean with "not required by default" that there should be a possibility to make bool field required if necessary? Or set bool field to non required somewhere else, e.g. [here](https://github.com/pydantic/FastUI/blob/13a9969885e7a7841e93c6bd5a898240fa7da35c/src/python-fastui/fastui/json_schema.py#L185)?

Reply:
@sergue1,

Yeah my apologies, my feedback was a bit confusing. Let me dig around this weekend and see if I can better understand how we might be able to fix this in a slightly different way!

Great work so far!